doc/user_guide.rst

```diff
@@ -13,16 +13,30 @@ Once you have downloaded a project you can run it locally or on AE.
 Run Locally
 ===========
 
-To run an example locally install anaconda-project and run the command
-defined in the anaconda-project file:
+To run an example locally first install anaconda-project.
 
 .. code:: bash
-
+   
    conda install anaconda-project=0.8.3
+
+Each project folder has a file `anaconda-project.yml`, which can be run as a command.
+
+.. code:: bash
+
    anaconda-project run
 
-Don’t want to use anaconda-project?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Running this command will will install the dependencies for the particular project and start a Bokeh server (e.g. it will end with the statement : `Bokeh app running at: http://localhost:5006/attractors_panel` ). This link can be used to see how the dashboard for the particular example works. 
```

Comment:
```suggestion
Running this command will install the dependencies for the particular project, then execute whatever the first command is. E.g. for a Bokeh or Panel dashboard, the default command could start a Bokeh server (e.g. it will end with a statement like: `Bokeh app running at: http://localhost:5006/attractors_panel` ). You can then open the given link to see the running dashboard. 
```